content/guides/advanced/how-to-request-optimal-compute.md

```diff
@@ -39,65 +39,67 @@ transaction. How do we come up with the number to use?
 
 The
 [simulateTransaction RPC method](https://solana.com/docs/rpc/http/simulatetransaction)
-will return the estimated compute units consumed given a transaction. Using this
-RPC method, you can calculate the compute units, set the compute units in your
-new transaction, and send it for an optimal result.
+will return the estimated compute units consumed given a transaction.
 
-An example function to get the compute units from a simulation looks as follows:
+The
+[Solana helpers npm package](https://www.npmjs.com/package/@solana-developers/helpers)
+includes
+[`getSimulationComputeUnits`](https://github.com/solana-developers/helpers?tab=readme-ov-file#get-simulated-compute-units-cus-for-transaction-instructions),
+a small function that uses `simulateTransaction` to calculate the compute units.
+You can then set the compute units in your new transaction, and send the new
+transaction for an optimal result.
 
-```javascript
-// import { ... } from "@solana/web3.js"
+```
+npm i @solana-developers/helpers
+```
 
-async function getSimulationUnits(
+The syntax is simply:
+
+```typescript
+getSimulationComputeUnits(
   connection: Connection,
-  instructions: TransactionInstruction[],
+  instructions: Array<TransactionInstruction>,
```

Comment:
Why change this to `Array`, while leaving `AddressLookupTableAccount[]`? I would just drop the `Array` or be consistent.